### ISSUE TITLE
Subtitles missing with MediaCodec Surface on Android

### ISSUE THREAD

**Issue by ghost** [xbmc/xbmc#15036]
### Describe the bug
It is an issue regarding no subtitles when using MediaCodec Surface on Android systems and the xbmc.Player() is in windowed mode.

## Expected Behavior
Subtitles should be shown.

## Actual Behavior
No subtitles are shown.

## Possible Fix
Disable MediaCodec Surface.

### To Reproduce
1. Run xbmc.Player().play in windowed mode.
2. Turn on MediaCodec Surface.
3. Start a playback.
4. Turn on subtitles.

### Debuglog

The debuglog can be found here:
https://paste.ubuntu.com/p/H5S5MFkBKt/

### Your Environment
Used Operating system:
 - [x] Android

 - Operating system version/name:
 - Kodi version: All versions of Kodi have this problem.

**Comment by iz8mbw** [comment]
Did you tested last nightly?
May be this already committed https://github.com/xbmc/xbmc/pull/15027 solve that issue

**Comment by M89SE** [comment]
@iz8mbw I will check it out and report back!

**Comment by M89SE** [comment]
Nope, the error persists.

**Comment by M89SE** [comment]
Do I need to provide any more information about the issue?

**Comment by MartijnKaijser** [comment]
Can you try with latest nightly?

**Comment by Mariusz89B** [comment]
Sure I will let you know tomorrow.

**Comment by Mariusz89B** [comment]
> Can you try with latest nightly?

Subtitles are still missing.

**Comment by MartijnKaijser** [comment]
@peak3d any idea?

**Comment by MartijnKaijser** [comment]
@Mariusz89B make sure you check all settings like position and if they are actually enabled.

**Comment by Mariusz89B** [comment]
> @Mariusz89B make sure you check all settings like position and if they are actually enabled.

Yep I have double checked, when turning of MediaCodec Surface the subs are working fine.
